draft-ietf-quic-transport.md

```diff
@@ -1055,46 +1112,50 @@ The fields in the WINDOW_UPDATE frame are as follows:
   flow control, the cumulative number of bytes which can be sent on all
   currently open streams.
 
-## BLOCKED Frame
+## BLOCKED Frame {#frame-blocked}
 
-A sender sends a BLOCKED frame when it is ready to send data (and has data to
-send), but is currently flow control blocked.  BLOCKED frames are purely
-informational frames, but extremely useful for debugging purposes.  A receiver
-of a BLOCKED frame should simply discard it (after possibly printing a helpful
-log message).  The frame is as follows:
+A sender sends a BLOCKED frame (type=0x05) when it is ready to send data (and 
+has data to send), but is currently flow control blocked. BLOCKED frames are 
+purely informational frames, but extremely useful for debugging purposes. A 
+receiver of a BLOCKED frame should simply discard it (after possibly printing a 
+helpful log message). The frame is as follows: 
 
 ~~~
-   +------------------------------+
-   |  Type(8)  |  Stream ID (32)  |
-   +------------------------------+
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                        Stream ID (32)                         |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ~~~
 
-The fields in the BLOCKED frame are as follows:
-
-* Frame Type: The Frame Type byte is an 8-bit value that must be set to 0x05
-  indicating that this is a BLOCKED frame.
+The BLOCKED frame payload contains a single field:
```

Comment:
Now that I understand what you were going for, I think the use of "payload" is perfectly accurate, but adds an unnecessary level of abstraction: we'll end up referring to "fields in the BLOCKED frame" anyways. In addition, we lose the ability to use "payload" for stream data that's carried within a STREAM frame. I'd prefer that we simply called it the frame's fields instead of the frame's payload.